drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -642,14 +712,21 @@ def comparesourcelists(slNames,imgNames,plotGen,diffMode,verbose,debugMode):
     verbose : Boolean
         display verbose output?
 
+    log_level : int, optional
+        The desired level of verboseness in the log statements displayed on the screen and written to the .log file.
+        Default value is 'NOTSET'.
+
     Returns
     -------
     overallStatus : string
         "OK" if all tests were passed, or "FAILURE" if inconsistencies were found.
     """
```

Comment:
The string `ERROR` is also returned in the event no comparisons could be performed.